src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -598,6 +597,10 @@ public void afterSave(@NonNull SCMNavigatorOwner owner) {
         }
     }
 
+    GitHubWebHook getHook() {
```

Comment:
@stephenc are you fine with the events test? If so, I can remove the webhook stuff to push this forward and, when you later refactor the webhook code, I can review that part.